src/bin/utils/cmds.rs

```diff
@@ -84,6 +84,20 @@ pool is encrypted, setting this option has no effect on the prediction."),
                         .action(ArgAction::Append)
                         .help("Size of filesystem to be made for this pool. May be specified multiple times, one for each filesystem. Units are bytes. Must be at least 512 MiB and less than 4 PiB.")
                         .next_line_help(true)
+                     )
+                     .arg(
+                         Arg::new("integrity_tag_size")
+                         .long("integrity-tag-size")
+                         .num_args(1)
+                         .help("Size of the integrity checksums to be stored in the integrity metadata. The checksum size depends on the algorithm used for checksums. Units are bytes.")
+                         .next_line_help(true)
+                     )
+                     .arg(
+                         Arg::new("integrity_journal_size")
+                         .long("integrity-journal-size")
+                         .num_args(1)
+                         .help("Size of the integrity journal. Default is 128 MiB. Units are 4KiB blocks.")
```

Comment:
Better just keep everything as bytes. I told Vojtech we would just do that, so we might as well stick with it.